doc/format.md

```diff
@@ -131,6 +131,8 @@ like the ATT&CK list.
   - `examples` is a *required* list of references to samples that the rule should match.
 The linter verifies that each rule correctly fires on each sample referenced in a rule's `examples` list.
 These example files are stored in the [github.com/fireeye/capa-testfiles](https://github.com/fireeye/capa-testfiles) repository.
+Example references use file names without extensions.

```

Comment:
i dont think this is right: the linter hashes all the files it finds in the testdata directory and uses this to pick the right file. it may also use the literal filename (see the PMA examples). i don't think it strips the extension anywhere.

Reply:
https://github.com/fireeye/capa/blob/master/scripts/lint.py#L480-L484

Reply:
![image](https://user-images.githubusercontent.com/156560/103805359-b289b480-5010-11eb-9195-389554652d69.png)

Reply:
Oh, good catch!